samples/net/latmon/prj.conf

```diff
@@ -0,0 +1,28 @@
+# SPDX-License-Identifier: Apache-2.0
+# Copyright (c) 2025 Jorge Ramirez-Ortiz <jorge.ramirez@oss.qualcomm.com>
+
+CONFIG_POSIX_API=y
+
+# General config
+CONFIG_LOG=y
+CONFIG_GPIO=y
+CONFIG_REBOOT=y
+CONFIG_TICKLESS_KERNEL=y
+CONFIG_CONSOLE_SUBSYS=y
+CONFIG_CONSOLE_GETLINE=y
+CONFIG_NEWLIB_LIBC=y
+CONFIG_INIT_STACKS=y
```

Comment:
I believe only `CONFIG_GPIO=y` and `CONFIG_LOG=y` need to be explicitly enabled. other options are platform specific, and should have reasonable defaults (hello_world sure does not select CONFIG_TICKLESS_KERNEL=y or CONFIG_REBOOT=y for example)

Reply:
also zephyr is moving to picolibc :)

Reply:
all right, testing. thanks!